sample_packages/sample_python_package/src/example.py

```diff
@@ -0,0 +1,19 @@
+import http.client
+import json
+
+# Sends an HTTPS post request and prints out the response.
+def sendHTTPSPostRequest():
```

Comment:
Can we have this do two different post requests?

One that happens on import, vs one that happens on a function that needs to be called (to test @maxfisher-g's work on executing functions).

Reply:
Done.